.github/workflows/ci.yml

```
@@ -0,0 +1,35 @@
+---
+name: ci
```

```suggestion
name: Publish
```

And rename the file for `publish.yml`